processing/src/main/java/org/apache/carbondata/processing/loading/TableProcessingOperations.java

```diff
@@ -152,6 +123,41 @@ public static void deletePartialLoadDataIfExist(CarbonTable carbonTable,
     }
   }
 
+  public static HashMap<CarbonFile, String> getStaleSegments(LoadMetadataDetails[] details,
```

Comment:
reformat the code, please follow formatting in all the changes

Reply:
done